x-pack/test/security_solution_cypress/cypress/screens/timeline.ts

```diff
@@ -296,7 +296,9 @@ export const TIMELINE_DATE_PICKER_CONTAINER = getDataTestSubjectSelector(
 
 export const TIMELINE_FILTER_BADGE = `[data-test-subj^='timeline-filters-container'] [data-test-subj^="filter-badge"]`;
 
-export const NEW_TIMELINE_ACTION = getDataTestSubjectSelector('new-timeline-action');
+export const NEW_TIMELINE_ACTION = `${getDataTestSubjectSelector('flyoutBottomBar')} 
```

Comment:
could this be the result of a bad rebase? I thought we removed `settings-plus-in-circle` in the recent timeline cleanup. I can't find any instances of `settings-plus-in-circle` in the entire repository

Reply:
No, not a rebase, just needed to update it. I fixed a bunch of these tests prior to Jatin's pr going in and 2/3 of them were being built off of his PR, but not this one. Just missed the hard coded value here. Thx